docs/concepts/clustering.rst

```diff
@@ -151,8 +151,8 @@ across the cluster.
    versions 4.x and above `determine quorum size automatically
    <https://crate.io/docs/crate/howtos/en/latest/clustering/multi-node-setup.html#master-node-election>`_.
 
-A minimum number of nodes (referred as a *quorum*) needs to configured (using the
-`discovery.zen.minimum_master_nodes`_ setting) to ensure that in case of a
+A minimum number of nodes (referred as a *quorum*) needs to be configured
+to ensure that in case of a
```

Comment:
Why do we want to remove the mentioning of the `minimum_master_nodes` here? As stated above, this paragraph is only needed for CrateDB < 4 where it is important to know and understand the `minimum_master_nodes` setting.

Reply:
Or maybe we remove this section completely or replace it with the content described at https://crate.io/docs/crate/howtos/en/latest/clustering/multi-node-setup.html#master-node-election?

Reply:
I replaced the section with a note and a link, please take a look to see if it looks ok like this.